hack/install-controller-gen.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copyright The Shipwright Contributors
+# 
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# Installs controller-gen utility via "go get".
+#
+
+set -eu
+
+# controller-gen version
+CONTROLLER_GEN_VERSION="${CONTROLLER_GEN_VERSION:-v0.4.1}"
+
+if [ ! -f "${GOPATH}/bin/controller-gen" ] ; then
+    echo "# Installing controller-gen..."
+    pushd "$(mktemp -d)" >/dev/null 2>&1
+    go get sigs.k8s.io/controller-tools/cmd/controller-gen@"${CONTROLLER_GEN_VERSION}"
```

Comment:
Just found out that in this particular case we have to make sure that it uses Go Modules otherwise you cannot specify a specific version. Tested it in an empty Docker container and with the environment variable it works for me.
```suggestion
    GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@"${CONTROLLER_GEN_VERSION}"
```

Reply:
+1, just hit the same issue

Reply:
Thanks, @HeavyWombat , I added.